docs/md/dev-wgs.md

```diff
@@ -50,3 +52,67 @@ The `unmapped_bam_suffix`
 is almost always `.unmapped.bam`,
 so that is its default value
 unless it is specified.
+
+The `reference_fasta_prefix` can be used to override
+the [default value](https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L7) used by this module: 
+"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38"
+
+## Usage
+
+###Create Workload: `/api/v1/create`
+Creates a WFL workload. Before processing, confirm that the WFL and Cromwell service accounts have
+at least read access to the input files.
+```
+curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/create' \
+--header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+  "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+  "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+  "pipeline": "ExternalWholeGenomeReprocessing",
+  "project": "PO-1234",
+  "items": [{
+          "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+          "sample_name": "TestSample1234"
+          }]}'
+```
+
+###Start Workload: `/api/v1/start`
+Starts a Cromwell workflow for each item in the workload. If an output already exists in the output bucket for a
+particular input cram, WFL will not re-submit that workflow.
+
+```
+curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/start' \
+--header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+--header 'Content-Type: application/json' \
+--data-raw '[{"uuid": "74d96a04-fea7-4270-a02b-a319dae2dd5e"}]'
```

Comment:
Would it be possible to make `--data-raw` take a list of uuids instead of a list of objects each with a `uuid` key-value pair?

Reply:
@tbl3rd do you remember why the input is structured like this?